### DISCUSSION TITLE
蘋果、Google、微軟攜手，一年內我們將迎接「沒有密碼」的時代

### DISCUSSION THREAD

**Discussion by Rosancl** [1101-GS4719-C/Python-Community-GS4719-C#614] [General]
FIDO 聯盟致力推廣生物辨識、身分認證等技術的標準規範，盼消除現今傳統數字、文字密碼的使用。
因此，全新的「無密碼登錄」標準，即是提供端到端的無密碼登入選項，使用者可以用每天最常使用的設備進行驗證，像是以指紋或臉部的簡單驗證，或裝置 PIN 等方式登入，使用者可能會在一天內多次進行此類操作。與密碼與傳統的多因素（如以簡訊發送一次性密碼）相比，這種新做法更可以防範網路釣魚，登入安全性也將大幅躍升。

https://buzzorange.com/techorange/2022/05/06/password-bye-hi-passkey/

**Comment by sys511613** [comment]
這樣就不會有一直忘記密碼的問題了 感覺很方便

**Comment by cordydeng** [comment]
感覺可以提升登入時的安全性，但未來如果有像是複製人的出現，指紋或臉部辨識應該也沒什麼可信度了

**Comment by jcpan821** [comment]
超讚 太久沒用的密碼都會忘記

**Comment by 11timyyy** [comment]
對於使用者來說，安全性真的非常重要!

**Comment by yuhao112** [comment]
又是一項很方便的技術呢

**Comment by henrywen547** [comment]
密碼越設越多組，從腦中忘掉的也越來越多組。
好期待一年之後我們的不用密碼的時代

**Comment by Yao900108** [comment]
對於金魚腦如我的人，非常期待這個技術之後的發展。

**Comment by jason79461385** [comment]
現在網頁設定密碼標準都不一樣有些規定要有一個大寫字母 有些還需要特殊符號 真的蠻煩人的

**Comment by Lynn12378** [comment]
很讚ㄟ，保有安全性，又不用擔心會忘記密碼

**Comment by Jimmy911218** [comment]
對於常忘記密碼的人來說真是一大救星阿

**Comment by Hchejxnc** [comment]
現在有的密碼真的太多組了，常會忘記導致要重新設定。
有了這個後，這個煩惱就解決了~

**Comment by 110409538** [comment]
有了這項技術,就不用擔心忘記密碼了.

**Comment by Doreen331** [comment]
這樣就可以降低忘記密或是帳號被盜的風險了，但其實取得別人的指紋也不是不可能，臉部所也有失誤的機率（整形/雙胞胎），只能說每個方法都有風險，沒有完美的方法，我們只能選擇風險相對小的去使用

**Comment by evanchou1204** [comment]
真的一大堆帳號的密碼都不一樣，記到快瘋了...

**Comment by a45320614** [comment]
這種方法能改善傳統帳號密碼的一個大缺點，許多人都會將不同網站的帳號密碼都設的一模一樣，如果有心人士取得了其中一個網站的資料，就可以連帶擁有其他網站的權限!

**Reply by bonny123865** [reply]
真的!!像是信箱、臉書、手機應用程式註冊，感覺有可能一個外洩之後，全部都會有被盜用風險。

**Comment by Zoe06060606** [comment]
這樣感覺提升安全性也增加方便性了

**Comment by Pietra1226** [comment]
這樣很方便是真的，但感覺之後就是駭客會去試圖取得一個人的生物辨識或身分認證資料，如果真的被發現漏洞，可能造成的損失更為巨大。

**Comment by Kotori0501** [comment]
現在的指紋辨識真的很方便，可以不太需要記密碼，但有時可能因為感應不良而造成無法辨識指紋，所以還是要記下密碼比較保險一點

**Comment by dabbie109303501** [comment]
指紋辨識和密碼方式應該併行比較好，要是辨識不靈敏或是手指上有沾到一些東西影響辨識就不方便了

**Comment by Yunhans** [comment]
以後就不用背一堆密碼了
想新密碼也是很累

**Comment by yuxinlin93** [comment]
這樣真的很方便，每次登入或註冊各種網站，每一種網站對密碼的要求都不同，有時候要有大寫，有時候需要有特殊字元，有時候又要有數字，就算有一組自己常用的密碼，有時候還是會忘記，選忘記密碼重改密碼卻又說不能使用舊密碼，遇到這種情況真的會很想翻桌

**Comment by WEI-CHEN-CHUN** [comment]
生物辨識技術的正確率仍存在不確定，因此短時間不太可能完全捨棄傳統密碼的模式

**Comment by CPCpyt** [comment]
感覺讓隱私安全多了一層保障，希望在未來能看到普及的一天。

**Comment by ruthchen1020** [comment]
我覺得最好保密還是人腦耶，這個比人腦有風險，但我也是個會忘密碼的人，所以不重要的最好幫我記一記，我可以自己記重要的。

**Comment by zephyrxin01** [comment]
但生物辨識本身就有很多隱私性考量的問題，畢竟通常來說這對一個人來說是獨一無二的。我沒記錯的話印度政府的Aadhaar就是擁有所有人的指紋。至少對我來說，我覺得國家擁有所有人的指紋是一件滿可怕的事，立法上要怎麼歸屬這個生物特徵為隱私還有很大的討論空間。

**Comment by alstinlin** [comment]
我覺得與傳統的身份鑒定手段相比，生物特征識別的身份辨識技術有以下優點：不易遺忘或丟失，防偽性能好，不易偽造或被盜，隨身攜帶，隨時隨地可用。

**Comment by Tzuhsuannnnn** [comment]
如果未來能都使用生物識別或其他技術來達成無密碼登入，應該會很方便，也可以大大減少釣魚網站的技術。
我覺得這個網站講關於釣魚網站的資訊講得很好，如果大家又興趣的話可以去看看，讓自己不要掉到陷阱或是成為釣魚網站的幫兇。
https://www.informationsecurity.com.tw/article/article_detail.aspx?aid=6813

**Comment by ashleyy-huang** [comment]
如果未來能夠達成無密碼登入的話那實在是太方便了，現在不管在網路上的哪個網站都需要註冊登入，累積下來就有幾十個帳號密碼，這樣會很容易導致忘記自己當初設定了什麼，我也希望未來可以盡量簡化帳號密碼註冊，像是直接用google登入就是個不錯的機制

**Comment by larry40305** [comment]
我覺得生物辨識的精確度目前還需要發展，因為我覺得指紋之類的東西其實還算好取得。

**Comment by zhiron544** [comment]
我覺得這項技術很棒，可以減少密碼被破解或洩漏的風險，也可以節省記憶空間和時間，但我也希望它能兼容不同的裝置和平台，不要變成每個網站都要用不同的方式登入。

**Comment by ThanatosJun** [comment]
消除密碼鎖改成生物特徵辨識，此舉的確減少密碼過多讓人難記的小問題，但我認為此引申出地一個隱患，若特徵值損毀，那修改方案又如何？生物特徵辨識的管理中心安全系統也將成為攻擊的重心。

**Comment by Alex11392** [comment]
感覺能讓方便性大幅提升，畢竟打密碼真的是一件很麻煩的事。
